sklearn/metrics/tests/test_common.py

```diff
@@ -1045,7 +1046,7 @@ def test_multioutput_regression_invariance_to_dimension_shuffling(name):
         )
 
 
-@ignore_warnings
+@pytest.mark.filterwarnings("ignore::sklearn.exceptions.UndefinedMetricWarning")
```

Comment:
Seems odd to have "UndefinedMetric" as a warning and not an error lol

Reply:
The undefined metric warning is essentially the one emitted when there's a division error in the computation of a metric due to not having all labels in y_pred or y_true. I think the goal of the warning and not an error is to not break cross validation where this situation can happen quite often.